Applications/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ if (MODULE_Registration)
   endmacro()
 
   add_registration_command(register LibIO OPTIONAL LibLBFGS LibPointSet LibDeformable)
+  add_registration_command(create-sdm LibIO)
```

Comment:
This tool is certainly not a registration command. It does not perform any actual registration. It rather belongs in the transformation command group. Relating to my other comment, it should be an option of the ```init-dof``` command.